Updated domain setup steps + Usage modification (#1147)
WalletConnect/walletconnect-docs: docs/web3inbox/domain-setup.mdx

<CloudBanner />

1. In [WalletConnect Cloud](https://cloud.walletconnect.com) navigate to the Explorer tab of your project.

   <img className="image-margin" src={useBaseUrl('/img/w3i/explorer.png')} />

> 2. Fill the required fields and click on the "Save" button. Over here, the Web App field is the domain of your app that you will use to publish notifications.

Reviewer: This can be done from the APIs tab, they don't need to use Explorer. Can we avoid the Explorer tab?

Author: under dapp information you mean?

Author: yep sure, that seems like its much easier to do, thanks!

   <img className="image-margin" src={useBaseUrl('/img/w3i/explorer-url.png')} />

3. Navigate to the APIs tab of your project.

   <img className="image-margin" src={useBaseUrl('/img/w3i/1.png')} />

4. Take note of your new Notify API Secret. You will need this to publish notifications. This secret
   allows publishing notifications to any account subscribed to your app, so should not be published
   and should only be used by your app backend.
   <img className="image-margin" src={useBaseUrl('/img/w3i/3.png')} />
5. Next, you will need to host `did.json` file on this page at the `/.well-known/` directory of your public URL.
   - Download `did.json` (click on `DID:WEB` section to show the download button ) and host it at, for example, `https://app.example.com/.well-known/did.json`
     <img className="image-margin" src={useBaseUrl('/img/w3i/did.png')} />

> 6. Once these two files have been deployed, switch on the toggle switch in the Notify API section of your WalletConnect Cloud project. If the files are correctly hosted, you should see a success toast: "Notify configuration successfully verified"

Reviewer: ```suggestion
6. Once this file has been deployed, switch on the toggle switch in the Notify API section of your WalletConnect Cloud project. If the files are correctly hosted, you should see a success toast: "Notify configuration successfully verified"
```
